elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -559,6 +559,16 @@ defmodule Domain.Config.Definitions do
   defconfig(:docker_registry, :string, default: "ghcr.io/firezone")
   defconfig(:api_url_override, :string, default: nil)
 
+  ##############################################
+  ## Sign Up
+  ##############################################
+
+  @doc """
+  String of comma separated email domains allowed to signup regardless of signup enabled state.
+  If signups are enabled, this list is ignored.
+  """
+  defconfig(:sign_up_allow_list, :string, default: "")
```

Comment:
Just a naming nitpick:
```suggestion
  defconfig(:sign_up_allowed_domains, :string, default: "")
```

Reply:
I'm curious what draws you to the name you suggested?  I actually had that originally, but changed it to provide more specificity about what the config variable should contain.  I'm good to use either one, just always interested in other people's thought process on variable naming.

Reply:
I think ideally we want the name to hint the type (which `sign_up_allow_list` does), or value (which does `sign_up_allowed_domains`). I would slightly lean towards `sign_up_allowed_domains` because `s` in the end kinda tells it's a list and also you know what you need to put into it, while `sign_up_allow_list` doesn't tell if you list emails, or domains, or something else

Reply:
🤦‍♂️ my brain switched the suggestion diff and I thought I had already put your suggestion in and you were suggesting `sign_up_allow_list`.  Sorry, I clearly need some sleep.  Everything you mentioned makes perfect sense.  I'll get this updated.